src/js/iqeEnablement.js

```diff
@@ -62,6 +62,13 @@ export default {
 
         return xhrResults.length > 0 || fetchResults.length > 0;
     },
+    isPageSafe: () => {
```

Comment:
No need to return `false`/`true` we can just return the value from `querySelector().length > 0`
```JS
isPageSafe: () => document.querySelectorAll('[data-ouia-safe=false]').length > 0
```